HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -18,6 +17,16 @@
 #                     pset.minGoodStripCharge = cms.PSet(refToPSet_ = cms.string('HLTSiStripClusterChargeCutNone'))
 #     return process
 
+# Matching ECAL selective readout in particle flow, need a new input with online Selective Readout Flags
+def customiseFor17794(process):
+     for edproducer in producers_by_type(process, "PFRecHitProducer"):
+          if hasattr(edproducer,'producers'):
+               for pset in edproducer.producers:
+                    if (pset.name == 'PFEBRecHitCreator' or pset.name == 'PFEERecHitCreator'):
+                         if not hasattr(pset,'srFlags'):
+                              pset.srFlags = cms.InputTag('hltEcalDigis')
```

Comment:
@slava77 @davidlange6 so should I add fillDescriptions in the producer? Can you point me to the domcumentation on how to do it?